guides/getting-started.md

```diff
@@ -9,22 +9,22 @@ The library uses Angular 5, if you have a previous version of angular see [the u
 :::
 
 :::
-##### Installing npm module
+##### Setting up project
 #### Step 1. Install npm module
 
-```BASH
+``` BASH
 npm i --save @healthcatalyst/cashmere
 ```
 
-#### Install peer dependencies (optional)
+#### Install peer dependencies on [popper.js](https://popper.js.org/), [font awesome](https://fontawesome.com), and [open sans](https://fonts.google.com/specimen/Open+Sans) (optional)
```

Comment:
Not sure if "optional" applies to the whole step or just to Open Sans
If if applies to the whole step, maybe put `(Optional)` at the beginning before `Install peer dependencies...`. This is how I changed it on Step 4 (commit coming soon).